### DISCUSSION TITLE
frp穿透内网的alist，域名访问报错System error: Error: [Hope UI]

### DISCUSSION THREAD

**Discussion by 825641748** [AlistGo/alist#5539] [Q&A]
使用frp穿透内网使用docker安装的小雅alist，域名去访问页面提示：System error: Error: [Hope UI]: useColorMode must be used within a HopeProvider，局域网内访问一切正常，请教一下这个问题如何解决？
内网页面
![image](https://github.com/alist-org/alist/assets/65830755/8a6f53e5-9659-4b6a-90b9-87989c53d51c)
外网页面
![image](https://github.com/alist-org/alist/assets/65830755/e9b88461-a861-4d76-afbd-f00fca67a4e2)
浏览器的报错信息
![image](https://github.com/alist-org/alist/assets/65830755/b1d668e2-beed-4d5e-b31d-6ce676d55dbe)
看起来像是HopeUI导致的问题

**Comment by zhoujinshi** [comment]
nginx配置 这样就可以了

location ~ / {
    ...
}

使用CDN加载远程资源就会报你这样的错误

**Comment by WorldCiao** [comment]
我也遇到了这个问题，把config.json里的cdn配置删除即可。
